lib/manageiq/release/mirror_helper.rb

```diff
@@ -0,0 +1,175 @@
+class String
+  def red;    "\e[31m#{self}\e[0m" end
+  def green;  "\e[32m#{self}\e[0m" end
+  def yellow; "\e[33m#{self}\e[0m" end
+  def cyan;   "\e[36m#{self}\e[0m" end
+  def bold;   "\e[1m#{self}\e[22m" end
+end
+
+require 'config'
+Config.load_and_set_settings(
+  Dir.glob(ManageIQ::Release::CONFIG_DIR.join("mirror*.yml")).sort.reverse
+)
```

Comment:
Not a fan of these things happening at the `lib/manageiq/release/mirror_helper.rb` level as just requiring this file ends up setting it globally (i.e. even in unrelated stuff).  IMO mirror_helper.rb should just have the module and then some bin script does the config setup (or it's passed in as a parameter and isolated to the mirrorheper instead of globally changing things)

Reply:
I switched to a class, so now they are done on initialize.  This changes the interface a little bit, but I think it's for the best.